src/Uno.UI/UI/Xaml/Media/MatrixTransform.cs

```diff
@@ -31,5 +31,7 @@ public Matrix Matrix
 				"Matrix", typeof(Matrix),
 				typeof(MatrixTransform),
 				new FrameworkPropertyMetadata(Matrix.Identity, NotifyChangedCallback));
+
+		public override string ToString() => Matrix.IsIdentity ? "<Identity>" : Matrix.ToString();
```

Comment:
Usually I prefer to not route the `.ToString()` to another type without indicate that the target type is wrapped (in a `Tranform` here)